### DISCUSSION TITLE
Can't Import changes from Git Repository

### DISCUSSION THREAD

**Discussion by bluksmat** [requarks/wiki#3564] [Error / Bug Report]
**Describe the bug**
Any changes made in repository, which are even in my local "/var/wiki/data/repo" are not imported/updated nor visible on Website. I tryed everything. Starting from clicking any possible button (including "Import Everything" from Git), had Bi-Directional as with "Pull from target". I have my changes on local repo but I can't make them active/visible on website (cache).

Tried:
Manual pull git - ok, all green, no effect
Click all Buttons under - Git nor Local, all green, no effect
remove files in "var/wiki/data/cache" ... no effect, they just re create without changes made in repo
Repeat all steaps and restart wiki (systemctl restart wiki) whenever possible... notching changed.

Note: Any changes made in Wiki Web Editor as correctly pushed to GitHub and local repo, just any changed in that local repo are not imported to actual website.

**To Reproduce**
1. Setup Git Repo. In short, all green, and make sure you can push/pull repo from to git
1. Make any page and fill it with any Content
2. Sync Git
3. Use any other PC/ Repo access point. Make change to page .md file (did manually updated date time)
4. Push Changes to the repo.
5. Go to your wiki web and sync Git (green light here).
6. Import Everything
7. Check actual Page which you modified (no diffrence - local repo have correct modified files)

**Expected behavior**
After at least clicking Import Everything, Pages should be updated with any modification made Page .md (or any other type) files. 
**Screenshots**
//No reason

**Host Info (please complete the following information):**
 - OS: Ubuntu 16.04
 - Wiki.js version: 2.0.0-beta.230 (and all previous)
 - Database engine: mySQL 15.1 -> 10.2.20-MariaDB

**Additional context**
//None

[a comment by an automated account is not shown]

**Comment by diabeeto076** [comment]
Git isn't working for me at all.

Even with the following entry in the logs:

```
wiki_1  | 2019-10-08T11:02:34.147Z [MASTER] info: Syncing with storage target git...
wiki_1  | 2019-10-08T11:02:34.155Z [MASTER] info: (STORAGE/GIT) Performing pull rebase from origin on branch master...
wiki_1  | 2019-10-08T11:02:34.263Z [MASTER] info: (STORAGE/GIT) Performing push to origin on branch master...
wiki_1  | 2019-10-08T11:02:34.354Z [MASTER] info: Syncing with storage target git: [ COMPLETED ]
wiki_1  | 2019-10-08T11:02:48.853Z [MASTER] info: Purging orphaned upload files...
wiki_1  | 2019-10-08T11:02:48.854Z [MASTER] info: Purging orphaned upload files: [ COMPLETED
```

Only the master branch is pulled into ./data/repo but nothing is imported/exported and/or pushed.
I've set the config to bidirectional

**Comment by AngryAnt** [comment]
Observing similar behaviour: Running bidirectional storage, but only export from database to git seems to have any effect. Version 2.0.12
